EazyE2E.Console/EazyE2E.Console.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Extracting the console portion of the app out to a console project and changed the EazyE2E projecto to type ClassLibrary.